docs/index.rst

```diff
@@ -13,21 +13,28 @@ Integrations Core API docs
 The Datadog Check Toolkit
 -------------------------
 
-The :ref:`datadog_checks` toolkit provides a set of functionalities used by any Agent based Integration:
+The :ref:`datadog_checks_base` package provides a set of functionalities used by any Agent based Integration:
 
-* :meth:`datadog_checks.checks.base.AgentCheck`, the base class that every Agent-based Integration is derived from.
+* :class:`~datadog_checks.base.checks.base.AgentCheck`, the base class that every Agent-based Integration is derived from.
 * :ref:`Prometheus <prometheus>` and :ref:`OpenMetrics <openmetrics>` facilities.
 * Boilerplate code implementing common operations.
 * A testing framework that can be used independently of the Agent.
 
+The :ref:`datadog_checks_dev` package provides tools to ease development and testing:
```

Comment:
```suggestion
The :ref:`datadog_checks_dev` package provides tools for use during development and testing:
```

"Ease" suggests that dev and testing are a negative condition that needs to be treated with painkillers or something. :)